package.json

```diff
@@ -29,6 +29,7 @@
     "lib": "lib"
   },
   "dependencies": {
+    "mailgun-js": "^0.7.7",
```

Comment:
Ok, should i take it out and add the requirement to the readme?
This is my first contribution to a project, if you dont mind guiding me towards what to do next that would be awesome

The error seems to be exactly that: the dependency missing from the project.